python-package/setup.py

```diff
@@ -204,7 +216,7 @@ def compile_cpp(use_mingw=False, use_gpu=False, use_cuda=False, use_mpi=False,
 
 class CustomInstallLib(install_lib):
 
-    def install(self):
+    def install(self) -> None:
```

Comment:
This one returns a list of strings (see `return outfiles` a few lines below).

```suggestion
    def install(self) -> List[str]:
```

Reply:
Thanks for pointing out my mistake! I applied your suggestion. Is there any thing more to change?